apps/emqx_authn/src/simple_authn/emqx_authn_pgsql.erl

```diff
@@ -54,9 +54,9 @@ fields(?CONF_NS) ->
     , {backend, emqx_authn_schema:backend(postgresql)}
     , {password_hash_algorithm, fun emqx_authn_password_hashing:type_ro/1}
     , {query,                   fun query/1}
-    ] ++ emqx_authn_schema:common_fields()
-    ++ emqx_connector_schema_lib:relational_db_fields()
-    ++ emqx_connector_schema_lib:ssl_fields().
+    ] ++
+    emqx_authn_schema:common_fields() ++
```

Comment:
I find `emqx_connector_schema_lib:relational_db_fields() ++ emqx_connector_schema_lib:ssl_fields()`  always appear at the same place. maybe we should export a function call `emqx_connector_schema_lib:relational_db_with_ssl_fields()`.
```erlang

```

Reply:
They are two different reusable part fields for DB connector.
Maybe shouldn't in single function.